src/Routes.tsx

```diff
@@ -22,11 +22,10 @@ function MainRoutes() {
           <Route path="/org-login" element={<ProtectedRoutes><Orglogin /></ProtectedRoutes>} />
           <Route path="/pricing" element={<Pricing />} />
           <Route path="/pricing-form" element={<Pay />} />
-          <Route path="/pricing-form" element={<Pay />} />
-          <Route path="/login-admin" element={<ProtectedRoutes><AdminLogin /></ProtectedRoutes>} />
+          <Route path="/signin" element={<AdminLogin />} />
           <Route path="/" element={<Perfomancetraineetable />} />
-          <Route path="/register-organization" element={<OrgRegister />} />
-          <Route path="/org-login" element={<Orglogin />} />
+          {/* <Route path="/register-organization" element={<OrgRegister />} /> */}
+          {/* <Route path="/org-login" element={<Orglogin />} /> */}
```

Comment:
Especially change this route of `org-login`
It is so hard to understand and even know the purpose since we already have a login and all the page does is checking if the organization exists

Reply:
What should be changed about it? Do you suggest to remove it completely?